sparkles/core.py

```diff
@@ -902,6 +903,22 @@ def check_acq_p2(self):
         elif P2 < P2_lim + 1:
             self.add_message('warning', f'P2: {P2:.2f} less than {P2_lim + 1} for {obs_type}')
 
+    def check_include_exclude(self):
+        """Check for included or excluded guide or acq stars (info)
+
+        """
+        call_args = self.call_args
+        for typ in ('acq', 'guide', 'fid'):
+            for action in ('include', 'exclude'):
+                ids = call_args.get(f'{action}_ids_{typ}')
+                if ids is not None:
+                    msg = f'{action}d {typ} ID(s): {ids}'
+                    halfws = call_args.get(f'{action}_halfws_{typ}')
```

Comment:
This is a little confusing since the only allowed type is include_haflws_acq .  Not sure what is won by having it in the loop.

Reply:
Ah.  I see, you are winning getting the message in a single line.  That's fine.

Reply:
The first draft explicitly only checked for `include_halfws_acq`, but the code logic was more complicated so I just chopped it down.